### ISSUE TITLE
The training log of DeiT III

### ISSUE THREAD

**Issue by tgxs002** [facebookresearch/deit#160]
Hi, I'm trying to reproduce the base model of DeiT III on ImageNet-1k with the suggested hyper-parameters. By running:
```
python run_with_submitit.py --model deit_base_patch16_LS --data-path /path/to/imagenet --batch 256 --lr 3e-3 --epochs 800 --weight-decay 0.05 --sched cosine --input-size 192 --reprob 0.0 --node 1 --gpu 8 --smoothing 0.0 --warmup-epochs 5 --drop 0.0 --nb-classes 1000 --seed 0 --opt fusedlamb --warmup-lr 1e-6 --mixup .8 --drop-path 0.2 --cutmix 1.0 --unscale-lr --repeated-aug --bce-loss  --color-jitter 0.3 --ThreeAugment
```
Will the training log be available? Or is it ok to share the accuracy on 1/2, 1/4 of the total schedule?

Thanks!

**Comment by TouvronHugo** [comment]
Hi @tgxs002,
Thanks for your message.
I'm trying to release the logs next week. Until then, here is the accuracy I have for the ViT-B:
- For the training we have the following accuracy:
epoch 0: 0.1
epoch 5: 2.838
epoch 10: 10.308
epoch 15: 17.292
epoch 20: 25.55
epoch 25: 33.436
epoch 30: 40.38
epoch 35: 45.446
epoch 40: 50.766
epoch 45: 54.162
epoch 50: 56.572
epoch 55: 58.8
epoch 60: 60.496
epoch 65: 61.994
epoch 70: 62.904
epoch 75: 64.06
epoch 80: 65.224
epoch 85: 65.718
epoch 90: 66.012
epoch 95: 66.984
epoch 100: 67.198
epoch 105: 67.864
epoch 110: 68.098
epoch 115: 68.568
epoch 120: 69.208
epoch 125: 69.448
epoch 130: 69.286
epoch 135: 69.568
epoch 140: 70.282
epoch 145: 70.396
epoch 150: 70.46
epoch 155: 70.844
epoch 160: 70.852
epoch 165: 71.33
epoch 170: 71.56
epoch 175: 71.784
epoch 180: 71.798
epoch 185: 72.162
epoch 190: 72.14
epoch 195: 72.082
epoch 200: 72.978
epoch 205: 72.562
epoch 210: 72.714
epoch 215: 73.006
epoch 220: 73.02
epoch 225: 73.466
epoch 230: 73.62
epoch 235: 73.656
epoch 240: 73.754
epoch 245: 73.718
epoch 250: 73.806
epoch 255: 74.182
epoch 260: 74.276
epoch 265: 74.592
epoch 270: 74.752
epoch 275: 74.86
epoch 280: 74.88
epoch 285: 75.044
epoch 290: 75.154
epoch 295: 75.482
epoch 300: 75.332
epoch 305: 75.548
epoch 310: 75.79
epoch 315: 75.832
epoch 320: 76.11
epoch 325: 76.27
epoch 330: 76.102
epoch 335: 76.556
epoch 340: 76.468
epoch 345: 76.514
epoch 350: 76.84
epoch 355: 77.128
epoch 360: 77.1
epoch 365: 76.99
epoch 370: 77.588
epoch 375: 77.336
epoch 380: 77.444
epoch 385: 77.924
epoch 390: 77.766
epoch 395: 77.756
epoch 400: 78.004
epoch 405: 78.238
epoch 410: 78.376
epoch 415: 78.45
epoch 420: 78.62
epoch 425: 78.628
epoch 430: 78.828
epoch 435: 78.92
epoch 440: 78.902
epoch 445: 78.994
epoch 450: 79.086
epoch 455: 79.206
epoch 460: 79.422
epoch 465: 79.394
epoch 470: 79.556
epoch 475: 79.736
epoch 480: 79.69
epoch 485: 79.978
epoch 490: 80.108
epoch 495: 80.162
epoch 500: 80.23
epoch 505: 80.346
epoch 510: 80.378
epoch 515: 80.488
epoch 520: 80.638
epoch 525: 80.666
epoch 530: 80.694
epoch 535: 80.798
epoch 540: 80.866
epoch 545: 81.024
epoch 550: 81.266
epoch 555: 81.028
epoch 560: 81.27
epoch 565: 81.306
epoch 570: 81.384
epoch 575: 81.474
epoch 580: 81.42
epoch 585: 81.53
epoch 590: 81.578
epoch 595: 81.726
epoch 600: 81.75
epoch 605: 81.672
epoch 610: 81.86
epoch 615: 82.006
epoch 620: 81.926
epoch 625: 81.806
epoch 630: 81.968
epoch 635: 82.104
epoch 640: 82.184
epoch 645: 82.202
epoch 650: 82.262
epoch 655: 82.224
epoch 660: 82.256
epoch 665: 82.284
epoch 670: 82.38
epoch 675: 82.438
epoch 680: 82.498
epoch 685: 82.554
epoch 690: 82.552
epoch 695: 82.648
epoch 700: 82.548
epoch 705: 82.702
epoch 710: 82.64
epoch 715: 82.598
epoch 720: 82.7
epoch 725: 82.686
epoch 730: 82.718
epoch 735: 82.71
epoch 740: 82.748
epoch 745: 82.784
epoch 750: 82.736
epoch 755: 82.77
epoch 760: 82.808
epoch 765: 82.754
epoch 770: 82.766
epoch 775: 82.798
epoch 780: 82.784
epoch 785: 82.802
epoch 790: 82.858
epoch 795: 82.866
epoch 800: 82.82

- For the fine-tuning we have the following accuracy:
epoch 0:  83.448
epoch 5: 83.71
epoch 10: 83.788
epoch 15:  83.826
epoch 20: 83.802

Best,

Hugo

**Comment by tgxs002** [comment]
Thank you for your quick response!

**Comment by tgxs002** [comment]
Hi Hugo,

Do you mind also sharing the training log of DeiT III base on IN-21K? :)

Thanks!
Blakey

**Comment by tgxs002** [comment]
Thanks!

**Comment by TouvronHugo** [comment]
It's done

**Comment by TouvronHugo** [comment]
For the 21k logs the evaluations are calculated on the ImageNet-1k validation set by making the prediction on the 21k classes and considering that all predictions outside the 1k classes are false.
